id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -393,23 +387,21 @@ private void checkForDuplicates(Map<String, Object> requestMap, Errors errors) {
 		} catch (IllegalStateException e) {
 			mosipLogger.error(IdRepoSecurityManager.getUser(), ID_REPO, ID_REQUEST_VALIDATOR,
 					("checkForDuplicates " + "  " + e.getMessage()));
-			errors.rejectValue(REQUEST, INVALID_INPUT_PARAMETER.getErrorCode(),
-					String.format(INVALID_INPUT_PARAMETER.getErrorMessage(), DOCUMENTS + " - "
-							+ StringUtils.substringBefore(StringUtils.reverseDelimited(e.getMessage(), ' '), " ")));
+			errors.rejectValue(REQUEST, INVALID_INPUT_PARAMETER.getErrorCode(), String.format(
```

Comment:
Why do we have error messages related code like this in multiple places. These should be helper methods in the error object itself.

Reply:
@ramesh-n Errors is spring built in class and it requires error code and message as parameter and so we have sent like this